static/app/views/monitors/components/overviewTimeline/index.tsx

```diff
@@ -151,6 +151,7 @@ export function OverviewTimeline({monitorList}: Props) {
       </Header>
       <GridLineOverlay
         stickyCursor
+        allowZoom
```

Comment:
should this also be `allowZoom={!isLoading}`?

Reply:
For some reason this is causing problems with selection. Need to dig on this

Reply:
Going to follow up on this, don't think it needs to block to ship this